PREPARATION.md

```diff
@@ -1,168 +1,146 @@
+- Any wallet that supports [WalletConnect](https://explorer.walletconnect.com/?type=wallet&chains=eip155%3A1).
 
-### POW faucet
+Whichever wallet you choose, at some point, it will ask you to configure a password to unlock your wallet. Your password, however, is **not** the real key to your wallet! Instead, they use a [mnemonic phrase](https://www.coinbase.com/learn/crypto-basics/what-is-a-seed-phrase) to generate your wallet. We recommend you write it down and store it somewhere safe.
 
-1. Go to https://goerli-faucet.pk910.de/ (or search for Goerli POW faucet)
-2. Enter your wallet address, and start mining! **Note:** You need to mine for a certain amount of time because there is a minimum claim amount. Make sure you have the minimum claim amount before you claim your funds.
-3. When you've had enough, click "Stop mining and claim rewards."
-4. Verify that you are a human, and your funds should appear shortly.
+> **Note:** If someone has access to your seed phrase, they will be able to access your wallet, all the accounts within it, and, therefore, all the funds. Consequently, it is best to get rid of this wallet after the workshop and create a new one in a more secure environment. Only use this one for testing purposes.
 
-**Tip**: If you have the resources, you can mine faster by increasing the number of workers. You can increase this beyond what is initially shown as the max, i.e., if it says 5/5, you can still increase the max workers if your machine allows it.
+Metamask holds your private key in the browser's data store. A private key is a randomly-generated number that gives a user control over all data and funds linked with the corresponding Ethereum address, which is generated from the Keccak-256 hash of the public key. This includes access to contracts that authorize the address. The private key is used to create signatures proving data or funds' ownership in a transaction. Therefore, never share your private key with anyone, as they can access your account and all the funds!
 
-## Other faucets links
+## Testnets and Test Eth
 
-We recommend (but it is optional) that you try out other faucets as well, [this site](https://faucetlink.to/goerli) has plenty of faucets you can try out. Some might be down; others might require some verification method, such as a tweet.
+"Testnets" are Ethereum networks that protocol developers use to test smart contracts in a development or staging environment. They are similar to the mainnet but not connected to it. Because of this, you can test smart contracts and applications without spending real money.
 
-## 4 Clone this repo and install VSCode extensions
+You can run a local testnet using [Anvil](https://book.getfoundry.sh/anvil/#how-to-use-anvil) or use a public testnet like [Goerli](https://goerli.net/#about). During this workshop, you'll also learn how. to deploy a contract to OpenSea, which only supports public testnets, so you should familiarize yourself with that.
```

Comment:
Typo, dot should be removed

"... you'll also learn how. to deploy ... " => "... you'll also learn how to deploy ..."